setup.py

```diff
@@ -29,6 +29,9 @@
         'simplejson',
         'six>=1.4.0',
     ],
+    extras_require={
+        'uwsgi': ['uWSGI>=2.0.14'],
```

Comment:
`>` here makes sense to me.  by "backport" in uwsgi's sense it usually means they'll apply the patch to a *new* version in previous lines (such as 2.0.13.x).  Note that it's actually not possible (without surgery) to replace an existing package on public pypi with a new package with different code at the same version.  It forbids you from uploading a file that has already existed in the past